### ISSUE TITLE
Should add aria roles

### ISSUE THREAD

**Issue by roblevintennis** [julian-printemps/vue3-pagination#5]
**Describe the bug**
The aria roles `aria-disabled` `aria-current` and `aria-label`'s should be added so the component is inclusive and accessible to all.
